stations.csv

```diff
@@ -70497,3 +70497,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74638;Bari Santa Rita;bari-santa-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
 74639;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74639;
 74640;Torino Grosseto;torino-grosseto;;;45.10546665758956;7.666751284086174;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8300097;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74640;
+74641;Antequera Av;antequera-av;7102030;;37.02978;-4.56119;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;02030;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74641;
+74642;A Gudina-Porta Galicia;a-gudina-porta-galicia;7108251;;42.06367;-7.13436;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;08251;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74642;
```

Comment:
![image](https://github.com/trainline-eu/stations/assets/4478372/0c21f79e-283d-42f2-8501-156fb5de4e72)

```suggestion
74642;A Gudiña – Porta de Galicia;a-gudina-porta-de-galicia;7108251;;42.06367;-7.13436;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;08251;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74642;
```

Reply:
LOL I've missed this one...

Reply:
I was about to comment that this should be reviewed by somebody more local to Spain. Stations is a detective's game, locals have a head start.